documentation/src/main/asciidoc/user_guide/clustered_lock.adoc

```diff
@@ -13,7 +13,7 @@ include::topics/dependencies/clustered_locks.adoc[]
 
 === ClusteredLock Configuration
 
-Currently there is a single type of `ClusteredLock` supported : non reentrant, NODE ownership lock.
+Currently there are the following `OwnershipLevel`: `NODE` and `INSTANCE`. At this moment, only `NODE` is supported.
```

Comment:
"Currently" and "at this moment" mean the same thing :)

Reply:
I prefer the original doc

Reply:
The original makes more sense to me too.

Reply:
I am still confused with the original doc. Let me try to explain what I see
```java
public enum OwnershipLevel {
   NODE,
   INSTANCE
}
```
If I open the source code for `ClusteredLock` I saw a lot of methods that won't let me know what are `NODE` or `INSTANCE`.

Reply:
This is an original error. I added the NODE and INSTANCE, but only node sync is possible today